scope.go

```diff
@@ -375,7 +383,27 @@ func (s *scope) Histogram(name string, b Buckets) Histogram {
 	return val
 }
 
-func (s *scope) Tagged(tags map[string]string) Scope {
+func (s *scope) Tagged(tags map[string]string, tenancys ...string) Scope {
```

Comment:
We should probably put `Tenancy string` in `ScopeOptions` such that it must be specified during `Scope` instantiation (if at all), as tenancy is something that would be fundamental to the scope of all metrics emitted by a given instance of `Scope`. In that model, we can just inject the tenancy into the tags upon instantiation and have that carried on with all subsequent `Scope` operations.

If you have a use case that requires a `Scope` that spans multiple tenancies (whether a metric that applies to N tenancies, or having a single root `Scope` have disparately-tenancied leaf `Scope`s), we should set up a chat so that I can learn more about your specific goals. I'm not opposed to adding support for both modes (both root `Scope` and leaf `Scope`), but I suspect the former will be far more prevalent and I'd like to start there.

Reply:
Hi Matt, I will set up a zoom chat regarding to the design requirements. Thanks!